test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/ResourceWrapper.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.test.system.framework.services.kubernetes;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.introspect.VisibilityChecker;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+
+/**
+ * Wrapper class to deserializing the json file contents.
+ */
+@Slf4j
+@Getter
+@Setter
```

Comment:
Why @setter annotation is needed ?

Reply:
Internally When ObjectMapper maps JSON to POJOs, it inspects the setter methods. ObjectMapper by default maps a key for the JSON field with the setter method name. So we need to have @Setter method on each of the POJO classes.

Reply:
ok